src/test/regress/bin/normalize.sed

```diff
@@ -297,3 +297,8 @@ s/"Total Cost": [0-9].[0-9]+/"Total Cost": xxxx/g
 
 # normalize gpids
 s/(NOTICE:  issuing SET LOCAL application_name TO 'citus_rebalancer gpid=)[0-9]+/\1xxxxx/g
+
+# shard_rebalancer output, flaky numbers for total_bytes
+s/public( *)\|( *)tab(.*)\|(.*)\|( *)[0-9]+/public\1\|\2tab\3\|\4\|\5xxxxxxx/g
```

Comment:
Like said in another comment, lets remove this rule by using a better size query.